x-pack/plugins/alerting/docs/openapi/entrypoint.yaml

```diff
@@ -14,62 +14,62 @@ tags:
+  '/api/alerting/rule/{ruleId}/alert/{alertId}/_mute':
+    $ref: 'paths/api@alerting@rule@{ruleid}@alert@{alertid}@_mute.yaml'
+  '/api/alerting/rule/{ruleId}/alert/{alertId}/_unmute':
+    $ref: 'paths/api@alerting@rule@{ruleid}@alert@{alertid}@_unmute.yaml'
 # Deprecated APIs
-  '/s/{spaceId}/api/alerts/alert/{alertId}':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/_disable':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@_disable.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/_enable':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@_enable.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/_mute_all':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@_mute_all.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/_unmute_all':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@_unmute_all.yaml'
-  '/s/{spaceId}/api/alerts/alerts/_find':
-    $ref: 'paths/s@{spaceid}@api@alerts@_find.yaml'
-  '/s/{spaceId}/api/alerts/alerts/_health':
-    $ref: 'paths/s@{spaceid}@api@alerts@_health.yaml'
-  '/s/{spaceId}/api/alerts/alerts/list_alert_types':
-    $ref: 'paths/s@{spaceid}@api@alerts@list_alert_types.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_mute':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@alert_instance@{alertinstanceid}@_mute.yaml'
-  '/s/{spaceId}/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_unmute':
-    $ref: 'paths/s@{spaceid}@api@alerts@alert@{alertid}@alert_instance@{alertinstanceid}@_unmute.yaml'
+  '/api/alerts/alert/{alertId}':
+    $ref: 'paths/api@alerts@alert@{alertid}.yaml'
+  '/api/alerts/alert/{alertId}/_disable':
+    $ref: 'paths/api@alerts@alert@{alertid}@_disable.yaml'
+  '/api/alerts/alert/{alertId}/_enable':
+    $ref: 'paths/api@alerts@alert@{alertid}@_enable.yaml'
+  '/api/alerts/alert/{alertId}/_mute_all':
+    $ref: 'paths/api@alerts@alert@{alertid}@_mute_all.yaml'
+  '/api/alerts/alert/{alertId}/_unmute_all':
+    $ref: 'paths/api@alerts@alert@{alertid}@_unmute_all.yaml'
+  '/api/alerts/alerts/_find':
+    $ref: 'paths/api@alerts@_find.yaml'
+  '/api/alerts/alerts/_health':
+    $ref: 'paths/api@alerts@_health.yaml'
+  '/api/alerts/alerts/list_alert_types':
+    $ref: 'paths/api@alerts@list_alert_types.yaml'
+  '/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_mute':
+    $ref: 'paths/api@alerts@alert@{alertid}@alert_instance@{alertinstanceid}@_mute.yaml'
+  '/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_unmute':
+    $ref: 'paths/api@alerts@alert@{alertid}@alert_instance@{alertinstanceid}@_unmute.yaml'
 
-components:
-  securitySchemes:
-    basicAuth:
-      type: http
-      scheme: basic
-    apiKeyAuth:
-      type: apiKey
-      in: header
-      name: Authorization
-      description: 'e.g. Authorization: ApiKey base64AccessApiKey'
-security:
-  - basicAuth: []
-  - apiKeyAuth: []
+# components:
```

Comment:
Should we uncomment this?

Reply:
Since it's the same info as exists for all Kibana APIs, it results in redundant noise when they're all joined together. I could just delete it but for now I've commented it out.

Reply:
I see! If it is not used could we delete it? The reason is that a commented piece of code may confuse someone in the future and not sure what to do with it.